### PR TITLE
Update contribute page

### PR DESCRIPTION
**Pull request by WilcoFiers**
- Split out the contribute page
- Fix some TBD links
- Add info about Slack

Need for Final Call: none

### CODE REVIEW

File: pages/contribute/rule-feedback.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+Start giving feedback on issues:
+
+- See the [ideas and early drafts that might need input](https://github.com/act-rules/act-rules.github.io/issues)
+
+- Use the [Definition of "Done"](../design/definition-of-done) to guide your review
+- Learn more about [commenting on issues](#) (TBD)
+
+## Feedback and reviews for draft rules (Pull Requests on GitHub)
+
+Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
```
Comment (carlosapaduarte) [inline pages/contribute/rule-feedback.md:26]: This makes it look like that before starting a review, one has to "register" for it, when all that is required is to have a GitHub account.

File: pages/contribute/join.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: Join the Community Group
+---
+
+The best way to stay informed about the activities of ACT-Rules Community is to become a member.
+
+## Mailing list keeps you up to date
+
+By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if no feedback
```
Comment (carlosapaduarte) [inline pages/contribute/join.md:9]: This sentence feels unfinished.

**Review by carlosapaduarte — CHANGES_REQUESTED** [review]

File: pages/contribute.md [hunk 3]
```diff
@@ -2,9 +2,7 @@
 title: How to Contribute
 ---
 
-ACT-Rules Community is a [W3C](https://www.w3.org/) [Community Group](https://www.w3.org/community/) open to anyone interested in accessibility testing, manually or automated.
-
-We welcome contributions from everyone, and there are many ways to contribute, from GitHub issues and pull requests, to contribution to our monthly teleconferences.
+ACT-Rules Community is a [W3C](https://www.w3.org/) [Community Group](https://www.w3.org/community/) open to anyone interested in accessibility testing, manually or automated. We welcome contributions from everyone, and there are many ways to contribute, from GitHub issues and pull requests, to contribution to ours teleconferences.
```
Comment (EmmaJP) [inline pages/contribute.md:5]:
```suggestion
ACT-Rules Community is a [W3C](https://www.w3.org/) [Community Group](https://www.w3.org/community/) open to anyone interested in accessibility testing, manually or automated. We welcome contributions from everyone, and there are many ways to contribute, from GitHub issues and pull requests, to contribution to our teleconferences.
```

Spelling

File: pages/contribute/join.md (hunk 2 above)
Comment (EmmaJP) [inline pages/contribute/join.md:9]: Perhaps...

```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if there is no unresolved feedback.
```

Comment (EmmaJP) [reply pages/contribute/join.md:9]: Perhaps...

```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes that have been approved in a GitHub pull request, and will be merged if there is no unresolved feedback.
```

File: pages/contribute/join.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+## Join us on Slack
+
+The ACT-Rules community is active in the #act-rules channel on the W3C Community Slack. To join, you will need to [sign up for the W3C community Slack](https://w3ccommunity.slack.com/join/shared_invite/zt-drq1eer6-MlZgOJOWVAEV1UTp66ywAQ#/). Once you have signed up, you can join by visiting [w3ccommunity.slack.com](https://w3ccommunity.slack.com/).
+
+## Participate in our calls
+
+The Community Group has conference calls twice a month. The first call is (usually) on the **second Thursday of the month** from [16:00 to 17:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200813T16&p1=16&ah=1&am=00). The second monthly call is on every **fourth Thursday of the month** from [10:00 to 11:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200827T10&p1=16&ah=1&am=00).
```
Comment (EmmaJP) [inline pages/contribute/join.md:17]:
```suggestion
The Community Group has conference calls twice a month. Usually, the first call is on the **second Thursday of the month** from [16:00 to 17:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200813T16&p1=16&ah=1&am=00), and the second monthly call is on every **fourth Thursday of the month** from [10:00 to 11:00 Central European Time](https://www.timeanddate.com/worldclock/fixedtime.html?msg=ACT-R+call&iso=20200827T10&p1=16&ah=1&am=00).
```

File: pages/contribute/propose-rules.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Submit a draft for a rule
+
+If you have a ready rule draft, you can submit it as a [pull request](https://github.com/act-rules/act-rules.github.io/pulls) to go into the [review process](/pages/design/process/).
+
+Before contributing a new rule, we recommend you check its validity with several experienced accessibility auditors first. This helps you identify potential stumbling blocks early in the rule design.
+
+## Using GitHub
+
+GitHub has excellent documentation. If you are new to GitHub, please see their documentation on how to [sign up](https://help.github.com/en/github/getting-started-with-github/signing-up-for-github), [navigate](https://help.github.com/en/github/managing-files-in-a-repository/navigating-code-on-github), [edit files](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository) and [reviewing pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests). In addition to official GitHub documentation, there are a lot of excellent articles and videos on how to use GitHub.
```
Comment (EmmaJP) [inline pages/contribute/propose-rules.md:25]:
```suggestion
GitHub has excellent documentation. If you are new to GitHub, please see their documentation on how to [sign up](https://help.github.com/en/github/getting-started-with-github/signing-up-for-github), [navigate](https://help.github.com/en/github/managing-files-in-a-repository/navigating-code-on-github), [edit files](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository) and [review pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests). In addition to official GitHub documentation, there are a lot of excellent articles and videos on how to use GitHub.
```

File: pages/contribute/rule-feedback.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
+
+If you "watch" the repository, you will get notifications when changes are happening.
+
+## Feedback for ideas and early drafts (Issues on GitHub)
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:11]: Are algorithms the same as definitions? Even for someone fairly familiar with things this paragraph isn't very clear.

File: pages/contribute/propose-rules.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+Whether you have a single great idea for an accessibility rule or you work for an organization that wants to contribute a whole repository of rules, we welcome the contribution.
+
+When you submit a rule, we expect you to be responsible for the rule the whole way through the review process, including adjusting the rule as you receive feedback from others. It will take some time and effort, but the learning experience in it is great.
+
+If you are not ready to carry a rule all the way through the process, you can partner up with someone else who wants to do it. Partnering up with someone outside of your own organization for rule writing will sharpen the rule, which is why we often do that in our work.
+
+## Submit an idea for discussion
+
+For rule contributors with less GitHub experience, it is often easiest to start the rule design in a [GitHub issue](https://github.com/act-rules/act-rules.github.io/issues). This allows for easy editing and discussions with others, while the rule is taking shape.
```
Comment (EmmaJP) [inline pages/contribute/propose-rules.md:15]: Is there a template for suggesting a new rule?

File: pages/contribute/propose-rules.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+## Submit a draft for a rule
+
+If you have a ready rule draft, you can submit it as a [pull request](https://github.com/act-rules/act-rules.github.io/pulls) to go into the [review process](/pages/design/process/).
```
Comment (EmmaJP) [inline pages/contribute/propose-rules.md:19]: Is there a template that a draft rule should use? 

I'm aware there is, but nothing here mentions that or links to it.

File: pages/contribute/rule-feedback.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
+---
+title: Give feedback on rules
+---
+
+Feedback on rules is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:5]:
```suggestion
Feedback is encouraged on our open [GitHub repository](https://github.com/act-rules/act-rules.github.io), so you don't even have to be a member of the Community Group to contribute in this way.
```

I think the first paragraph on this page should cover all types of feedback, not only on rules.

File: pages/contribute/rule-feedback.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+If you "watch" the repository, you will get notifications when changes are happening.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:7]: It might be helpful if "watch" links to something that explains how to do that.

File: pages/contribute/rule-feedback.md (hunk 6 above)
Comment (EmmaJP) [reply pages/contribute/rule-feedback.md:11]:
```suggestion
Draft rules and definitions (shared terms across rules) that are currently GitHub issues can be anything from just a title to an almost finished rule. These issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
```

File: pages/contribute/rule-feedback.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+## Feedback for ideas and early drafts (Issues on GitHub)
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
+
+The top comment on an issue should hold the updated rule or algorithm, and underneath there will be a thread of comments and a history of actions related to the issue.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:13]:
```suggestion
The top comment on such an issue should hold the most up to date version of the draft rule or algorithm, and underneath there will be a thread of related comments and a history of actions.
```

File: pages/contribute/rule-feedback.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+Please feel free to add your own comments to issues.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:15]:
```suggestion
Please feel free to add your own comments to these issues.
```

File: pages/contribute/rule-feedback.md [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+Start giving feedback on issues:
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:17]:
```suggestion
To start giving such feedback:
```

File: pages/contribute/rule-feedback.md (hunk 1 above)
Comment (EmmaJP) [reply pages/contribute/rule-feedback.md:26]: Does someone need to be a member of the community group to review a PR, or is this still just GitHub permission levels?

Comment (EmmaJP) [reply pages/contribute/rule-feedback.md:26]:
```suggestion
Pull requests are rule drafts that are ready for peer reviews. You can assign yourself as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to review it. However, even if three people are already reviewing it, we can always use more eyes on things.
```

File: pages/contribute/rule-feedback.md [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+
+Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
+
+You can both add single comments to a pull request and do a full review.
+
+**Only approve a rule** if you feel confident (to the best of your knowledge) that this rule is 100% ready to be a published ACT rule. Please also refer to the [Definition of "Done"](../design/definition-of-done).
+
+If you for some reason are not so confident, you can always leave a comment for the things that you have an opinion about, without doing a full review.
+
+Start giving feedback and doing reviews for draft rules:
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:34]:
```suggestion
To start giving feedback and doing reviews for draft rules:
```

File: pages/contribute/rule-feedback.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+- See the [draft rules, algorithms and more that need more reviewers](https://github.com/act-rules/act-rules.github.io/pulls?q=is%3Aopen+is%3Apr+label%3A%22reviewer+wanted%22)
+- Use the [Definition of "Done"](../design/definition-of-done) to guide your review
+- Learn more about [reviewing and commenting on pull requests](#) (TBD)
+
+## Feedback/corrections for published rules
+
+We welcome any input for changes, also for already published rules. The published rules should be as precise as possible, limiting the number of potential false positives and false negatives. For this it is important to draw on as many tool and methodology implementations and expert opinions as possible.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:42]:
```suggestion
We also welcome any feedback and suggestions for changes to already published rules. The published rules should be as precise as possible, limiting the number of potential false positives and false negatives. For this, it is important to draw on as many tool and methodology implementations and expert opinions as possible.
```

File: pages/contribute/rule-feedback.md [hunk 16]
```diff
@@ -0,0 +1,49 @@
...
+
+Feedback and corrections for existing rules can target both on the sections of the rule itself (Applicability, Expectations, Accessibility Support, etc.) and the test cases, to e.g. expand the edge case coverage.
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:44]:
```suggestion
Feedback and corrections for existing rules can target any of the sections of the rule itself (Applicability, Expectations, Accessibility Support, etc.), as well as the test cases, for example to expand the edge case coverage.
```

File: pages/contribute/rule-feedback.md [hunk 17]
```diff
@@ -0,0 +1,49 @@
...
+
+You have several options for correcting existing rules:
+
+- **Open a discussion in an Issue:** If you want to open a discussion, you can [open a new issue on GitHub](https://github.com/act-rules/act-rules.github.io/issues) repository to discuss the issue with the existing rule. Learn more about [creating issues](#) (TBD).
```
Comment (EmmaJP) [inline pages/contribute/rule-feedback.md:48]:
```suggestion
- **Open a discussion in an Issue:** If you want to open a discussion, you can [open a new issue on the GitHub repository](https://github.com/act-rules/act-rules.github.io/issues) to discuss the issue with the existing rule. Learn more about [creating issues](#) (TBD).
```

Alternatively remove the word repository.

**Review by EmmaJP — DISMISSED** [review]
Have left several comments. Some are straight up spelling or grammar issues. Some are things that need to be clearer.

File: pages/contribute/join.md (hunk 2 above)
Comment (Jym77) [inline pages/contribute/join.md:9]: I'm not sure about the reference to Github here. As in, it might scare people away feeling they need to know what is Github and what is a pull request in order to join the CG.
```suggestion
By becoming a member you will be added to the mailing list, and you will receive all the latest updates, meeting invites, etc. You will receive "final call" e-mails. These are announcements of changes (new rules, correction to existing ones, …) that have been approved by at least 3 other members, and will be published once there is no unresolved feedback.
```

Maybe also add a rough estimate of the number of emails (2-3/week?) to avoid scaring people in fear of being "spammed" (the list here sounds like there is a lot happening on the ML, but it is not really the case).

File: pages/contribute/rule-feedback.md (hunk 10 above)
Comment (Jym77) [reply pages/contribute/rule-feedback.md:7]: It might actually be possible to have a link to watch automatically.
(when new repos are created in my org, Github auto-watch me and send me a link to un-watch; if link to un-watch exists, I assume link to watch should exist)

File: pages/contribute/rule-feedback.md (hunk 6 above)
Comment (Jym77) [reply pages/contribute/rule-feedback.md:11]: I'm also not sure what "Algorithm" refers to in our process 🤔

File: pages/contribute/rule-feedback.md [hunk 18]
```diff
@@ -0,0 +1,49 @@
...
+
+Rules and Definitions (shared terms across rules) that are GitHub Issues can be everything from just a title to a quite finished rule. The issues will be prefixed with either "Rule:" (or possibly "New Rule:") or "Algorithm".
+
+The top comment on an issue should hold the updated rule or algorithm, and underneath there will be a thread of comments and a history of actions related to the issue.
+
+Please feel free to add your own comments to issues.
+
+Start giving feedback on issues:
+
+- See the [ideas and early drafts that might need input](https://github.com/act-rules/act-rules.github.io/issues)
```
Comment (Jym77) [inline pages/contribute/rule-feedback.md:19]: Should we filter that on labels such as "rule idea" or "Should we proceed?" or some other?
Many issues are problem with existing rules and not ideas/early draft, and the feedback to give is quite different…

File: pages/contribute/rule-feedback.md [hunk 19]
```diff
@@ -0,0 +1,49 @@
...
+
+- Use the [Definition of "Done"](../design/definition-of-done) to guide your review
+- Learn more about [commenting on issues](#) (TBD)
+
+## Feedback and reviews for draft rules (Pull Requests on GitHub)
+
+Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
+
+You can both add single comments to a pull request and do a full review.
```
Comment (Jym77) [inline pages/contribute/rule-feedback.md:28]:
```suggestion
You can either add single comments to a pull request or do a full review.
```

File: pages/contribute/rule-feedback.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+**Only approve a rule** if you feel confident (to the best of your knowledge) that this rule is 100% ready to be a published ACT rule. Please also refer to the [Definition of "Done"](../design/definition-of-done).
+
+If you for some reason are not so confident, you can always leave a comment for the things that you have an opinion about, without doing a full review.
+
+Start giving feedback and doing reviews for draft rules:
+
+- See the [draft rules, algorithms and more that need more reviewers](https://github.com/act-rules/act-rules.github.io/pulls?q=is%3Aopen+is%3Apr+label%3A%22reviewer+wanted%22)
```
Comment (Jym77) [inline pages/contribute/rule-feedback.md:36]: Here also, I do not know what "algorithms" means in our context.

**Review by Jym77 — CHANGES_REQUESTED** [review]
Seconding existing suggestions and adding a few of my own.

File: pages/contribute/join.md (hunk 2 above)
Comment (EmmaJP) [reply pages/contribute/join.md:9]: That's a good suggestion. Although it may still be worth mentioning that those announced changes may be viewed on the Github without needing an account (which I think it true).

File: pages/contribute/rule-feedback.md (hunk 18 above)
Comment (EmmaJP) [reply pages/contribute/rule-feedback.md:19]: That would be good, if there is a generic filter for "ideas and early drafts", which is what this section covers. It would need to include definition drafts as well as rule ideas and drafts, and anything similar.

File: pages/contribute/rule-feedback.md (hunk 19 above)
Comment (EmmaJP) [reply pages/contribute/rule-feedback.md:28]: Except that you can do both. So perhaps "and/or".

File: pages/contribute/rule-feedback.md (hunk 6 above)
Comment (WilcoFiers) [reply pages/contribute/rule-feedback.md:11]: Yikes! Yeah "algorithm" is what we used to call definitions, when rules were would prescribe the exact procedure to follow.

**Review by nitedog — APPROVED** [review]
Love these changes. Several good suggestions from @EmmaJP and @Jym77 that I agree with.

File: pages/contribute/propose-rules.md (hunk 7 above)
Comment (nitedog) [reply pages/contribute/propose-rules.md:15]: Added to next section.

File: pages/contribute/propose-rules.md (hunk 8 above)
Comment (nitedog) [reply pages/contribute/propose-rules.md:19]: Added.

File: pages/contribute/rule-feedback.md (hunk 10 above)
Comment (nitedog) [reply pages/contribute/rule-feedback.md]: Added link to Propose Rules page.

File: pages/contribute/propose-rules.md [hunk 21]
```diff
@@ -0,0 +1,27 @@
...
+
+Before contributing a new rule, we recommend you check its validity with several experienced accessibility auditors first. This helps you identify potential stumbling blocks early in the rule design.
+
+## Using GitHub
+
+GitHub has excellent documentation. If you are new to GitHub, please see their documentation on how to [sign up](https://help.github.com/en/github/getting-started-with-github/signing-up-for-github), [navigate](https://help.github.com/en/github/managing-files-in-a-repository/navigating-code-on-github), [edit files](https://help.github.com/en/github/managing-files-in-a-repository/editing-files-in-another-users-repository) and [review pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests). In addition to official GitHub documentation, there are a lot of excellent articles and videos on how to use GitHub.
+
+For details details, see the [GitHub Guidelines for ACT-Rules](https://github.com/act-rules/act-rules.github.io/blob/develop/github-guidelines.md).
```
Comment (carlosapaduarte) [inline pages/contribute/propose-rules.md:27]:
```suggestion
For details, see the [GitHub Guidelines for ACT-Rules](https://github.com/act-rules/act-rules.github.io/blob/develop/github-guidelines.md).
```

**Review by carlosapaduarte — DISMISSED** [review]
Found an edit needed

File: pages/contribute/rule-feedback.md [hunk 22]
```diff
@@ -23,27 +23,27 @@ Start giving feedback on issues:
 
 ## Feedback and reviews for draft rules (Pull Requests on GitHub)
 
-Pull requests are rule drafts that are ready for peer reviews. You can sign up as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to sign up for reviewing it. But even if three people are already signed up, we can always use more eyes on it.
+Pull requests are rule drafts that are ready for peer review. You can assign yourself as a reviewer and follow the rule through iterations. A rule needs three approvals, and if a rule has the label "Reviewer wanted", we need more people to review it. However, even if three people are already reviewing it, we can always use more eyes on things.
```
Comment (Jym77) [inline pages/contribute/rule-feedback.md:26]: "assign yourself as a reviewer" is incorrect in Github lingo.
"assign" is used by Github for the person(s) in charge of the PR, reviewers are "requested". There has been several occurrences of people "assigning" other rather than "requesting review" from them. I'm a bit afraid this may drive non Github-savy people to click the "assign" button rather than the "request review" one. Especially, there is a Github button called "assign yourself" which is **not** what we want people to click in order to be reviewers…

assign-review


OTOH, I agree that "you can request a review from yourself" sounds plain wrong in regular language… 🤔 I'm not having a good suggestion here.

**Review by Jym77 — COMMENTED** [review]
Some tech lingo discrepancy. Not really blocking, I guess…

File: pages/contribute/rule-feedback.md (hunk 22 above)
Comment (nitedog) [reply pages/contribute/rule-feedback.md]: changed "assign yourself..." to "add yourself...", which should now be hopefully less confusing.

**Review by Jym77 — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]
